package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "neo4j-migrations-docs",
+  "version": "1.0.0",
```

Comment:
Can this be derived from the maven version / tag? This is not accurate.

Reply:
This package won't be published so the version does not really matter. Though, I need to add `private: true` to make sure that we won't inadvertently publish it: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#private